### ISSUE TITLE
checkstyle doesn't work with the prebuilt google_checks.xml

### ISSUE THREAD

**Issue by maseev** [checkstyle/checkstyle#2357]
Hello there!

We have run into a problem recently when checkstyle was refusing to work with the prebuilt google_checks rules. checkstyle passed all checks even though there were a lot of code style violations. After a short investigation we noticed that google_checks.xml contained the following line:

```xml
<property name="severity" value="warning"/>
```

We decided to move the google_checks.xml file from the jar to the project's resource folder, comment that line and point maven-checkstyle-plugin to this xml file. As a result, we got a bunch of errors which checkstyle found after the check goal.

Here's a maven plugin configuration that we're using with checkstyle plugin:

```xml
            <plugin>
                <groupId>org.apache.maven.plugins</groupId>
                <artifactId>maven-checkstyle-plugin</artifactId>
                <version>2.16</version>
                <executions>
                    <execution>
                        <id>checkstyle</id>
                        <phase>validate</phase>
                        <goals>
                            <goal>check</goal>
                        </goals>
                        <configuration>
                            <configLocation>src/main/resources/google_checks.xml</configLocation>
                            <failOnViolation>true</failOnViolation>
                        </configuration>
                    </execution>
                </executions>
                <dependencies>
                    <dependency>
                        <groupId>com.puppycrawl.tools</groupId>
                        <artifactId>checkstyle</artifactId>
                        <version>6.11.2</version>
                    </dependency>
                </dependencies>
            </plugin>
```

It seems that google_checks was broken from the beginning.

**Comment by romani** [comment]
first of all this Checkstyle and chekstyle-maven-plugin are different projects.....

1) please share error message or stacktrace or ....
2) please reproduce this with our CLI - http://checkstyle.sourceforge.net/cmdline.html#Download_and_Run

FYI:
we have bunch of ITs over goolge_checks.xml - https://github.com/checkstyle/checkstyle/blob/master/src/it/java/com/google/checkstyle/test/base/ConfigurationBuilder.java#L23

**Comment by maseev** [comment]
Couldn't reproduce this behaviour by using the CLI. It seems that the problem is on the maven-checkstyle-plugin's side.

**Comment by maseev** [comment]
I think I could figure this problem out. maven-checkstyle-plugin has the following property which enables the console output:

```xml
<consoleOutput>true</consoleOutput>
```

That's why I didn't see any output messages from the plugin. Because the google_checks.xml has the severity level = "warning" and it's less severe than "error".

The other thing is the severity level, which has a value "error" by default. The severity level can be tuned by setting the following property:

```xml
<violationSeverity>warning</violationSeverity> // info, warning, error
```

so that the build would be broken if any warning appears.

Sorry for the disturbance.
